os/drivers/lcd/mipi_lcd.c

```diff
@@ -436,6 +436,7 @@ FAR void lcd_put_logo(FAR struct lcd_dev_s *dev)
 	 * then filling it with logo data on specific index in while loop */
 	memset(lcd_logo_fullscreen_data, LCD_BLACK_VAL, CONFIG_LCD_XRES * CONFIG_LCD_YRES * 2);
 
+#if defined(CONFIG_LCD_LOGO)
```

Comment:
The function name is `lcd_put_logo`. But the conditional is added only here. 
Is that to print black screen when log is disabled?
If yes, how about renaming the function and the variables like lcd_init_xxxx?

Reply:
lcd put(or set0 initial screen function prints
1. logo with the logo config enabled
2. black screen with logo config disabled.

I think this makes sense in name (You already make that in operation, I mean in name)

Reply:
ok, I have changed the name of this function.